File: app/src/main/java/com/task/paginationlist/presentation/piclist/views/utils/ErrorHandler.java
package com.task.paginationlist.presentation.piclist.views.utils;

import android.content.Context;
import android.support.v7.app.AlertDialog;


public class ErrorHandler {

    public void handlerErrorMessage(Context context, String message) {
        new AlertDialog.Builder(context)
                .setMessage(message)
                .setCancelable(true)
                .show();
    }
}

File: app/src/main/java/com/task/paginationlist/presentation/piclist/views/utils/ErrorHandler.kt
package com.task.paginationlist.presentation.piclist.views.utils

import android.content.Context
import android.support.v7.app.AlertDialog


class ErrorHandler {

    fun handlerErrorMessage(context: Context, message: String) {
        AlertDialog.Builder(context)
                .setMessage(message)
                .setCancelable(true)
                .show()
    }
}
